Formula/t/tmux-sessionizer.rb

```diff
@@ -0,0 +1,48 @@
+class TmuxSessionizer < Formula
+  desc "Tool for opening git repositories as tmux sessions"
+  homepage "https://github.com/jrmoulton/tmux-sessionizer/"
+  url "https://github.com/jrmoulton/tmux-sessionizer/archive/refs/tags/v0.4.2.tar.gz"
+  sha256 "0f9369a045ebe181202fcf5f292bbdd836f25b47ca9da1702351a725693631f5"
+  license "MIT"
+
+  depends_on "pkg-config" => :build
+  depends_on "rust" => :build
+  depends_on "libgit2@1.7"
+  depends_on "libssh2"
+  depends_on "openssl@3"
+  uses_from_macos "zlib"
+
+  def install
+    # Ensure that the `openssl` crate picks up the intended library.
+    ENV["OPENSSL_DIR"] = Formula["openssl@3"].opt_prefix
+    ENV["OPENSSL_NO_VENDOR"] = "1"
+
+    ENV["LIBGIT2_NO_VENDOR"] = "1"
+    ENV["LIBSSH2_SYS_USE_PKG_CONFIG"] = "1"
+
+    system "cargo", "install", *std_cargo_args
+  end
+
+  def check_binary_linkage(binary, library)
+    binary.dynamically_linked_libraries.any? do |dll|
+      next false unless dll.start_with?(HOMEBREW_PREFIX.to_s)
+
+      File.realpath(dll) == File.realpath(library)
+    end
+  end
+
+  test do
+    assert_match "Configuration has been stored", shell_output("#{bin}/tms config -p /dev/null")
+    assert_match version.to_s, shell_output("#{bin}/tms --version")
```

Comment:
output of the version command looks like this
```suggestion
    assert_match "tmux-sessionizer ${version}", shell_output("#{bin}/tms --version")
```